modules/disk-uploader/pkg/image/image.go

```diff
@@ -16,7 +17,24 @@ import (
 	tar "kubevirt.io/containerdisks/pkg/build"
 )
 
-func Build(diskPath string) (v1.Image, error) {
+const (
+	defaultInstanceType = "INSTANCETYPE_KUBEVIRT_IO_DEFAULT_INSTANCETYPE"
+	defaultPreference   = "INSTANCETYPE_KUBEVIRT_IO_DEFAULT_PREFERENCE"
+)
+
+func DefaultConfig(labels map[string]string) v1.Config {
```

Comment:
@0xFelix @lyarwood Currently this function returns a config with default instancetype and preference, what would we do in case of custom instancetype or preference?

Reply:
@codingben custom in terms of something different being used by a `VirtualMachine` or `VirtualMachineSnapshot`? We could just override the defaults with the values used there but I'm not sure I entirely understand the flow here tbh.

Reply:
@lyarwood Can be there something else than `instancetype.kubevirt.io/default-instancetype` or `instancetype.kubevirt.io/default-preference`? not a default one?

Reply:
We also support `-kind` variants of both of these labels:

https://github.com/kubevirt/kubevirt/blob/0436bc52bf02c689190cc86889c3ec90b8bcfecf/staging/src/kubevirt.io/api/instancetype/register.go#L42-L47

Reply:
@lyarwood Thanks. I've modified it in this last [revision](https://github.com/kubevirt/kubevirt-tekton-tasks/compare/fdc80d265fe74f103a1ce3fe8e2442fc3366b58a..a3b9ff76d4161b586c65313e23d4743577f1829e).